code/data_cleaning.R

```diff
@@ -205,12 +205,53 @@ data <- data %>%
   ) %>%
   mutate(standardized_facility = ifelse(facility_name_title_case %in% c(
     "Ssam / Dhs", "Centru Saude Deho", "Ps Estado", "Postu Da Saude",
-    "Chc Centru Saude", "Js200921", "Hp", "Hp Deleco", "Hp Ekapu", "Ps Fatuquero",
+    "Chc Centru Saude", "Js200921", "Hp", "Hp Deleco", "Hp Ekapu",
     "Ps Vatunau", "Hp Lebutelo", "Posto Saude", "Cs Internamentu"
   ), NA, standardized_facility)) %>%
   relocate(standardized_facility, .after = facility) %>%
   left_join(standard_facility_names, by = c("standardized_facility" = "full_name"))
 
+# Create new column for CHCs & the health posts in their region (called "CHC_catchment)
+data <- data %>%
+  mutate(CHC_catchment = case_when(
```

Comment:
For all of this, you could just update region in the excel spreadsheet. Looks like I just missed Ermera? Then you can just keep the variable `region` but it'll be correct. We're not really using `region` for anything else, so mind as well update that instead of creating a new var with a lot of code!